core/vtk/ttkTriangulationRequest/ttkTriangulationRequest.h

```diff
@@ -87,58 +65,32 @@ class TTKTRIANGULATIONREQUEST_EXPORT ttkTriangulationRequest
   vtkSetMacro(KeepAllDataArrays, bool);
   vtkGetMacro(KeepAllDataArrays, bool);
 
-  vtkSetMacro(PeriodicBoundaryConditions, int);
-  vtkGetMacro(PeriodicBoundaryConditions, int);
-
-  int FillInputPortInformation(int port, vtkInformation *info) override {
-
-    switch(port) {
-      case 0:
-        info->Set(vtkDataObject::DATA_TYPE_NAME(), "vtkDataSet");
-        break;
-
-      default:
-        break;
-    }
-
-    return 1;
-  }
-
-  int FillOutputPortInformation(int port, vtkInformation *info) override {
-
-    switch(port) {
-      case 0:
-        info->Set(vtkDataObject::DATA_TYPE_NAME(), "vtkUnstructuredGrid");
-        break;
-
-      default:
-        break;
-    }
-
-    return 1;
-  }
-
 protected:
   ttkTriangulationRequest() {
-    UseAllCores = true;
     SimplexType = 0;
     SimplexIdentifier = 0;
     RequestType = 0;
     KeepAllDataArrays = true;
-    PeriodicBoundaryConditions = false;
 
     SetNumberOfInputPorts(1);
     SetNumberOfOutputPorts(1);
+
+    this->setDebugMsgPrefix("TriangulationRequest");
   }
 
   ~ttkTriangulationRequest() override{};
 
-  TTK_SETUP();
+  int FillInputPortInformation(int port, vtkInformation *info) override;
+
+  int FillOutputPortInformation(int port, vtkInformation *info) override;
+
+  int RequestData(vtkInformation *request,
+                  vtkInformationVector **inputVector,
+                  vtkInformationVector *outputVector) override;
 
 private:
   int SimplexType;
```

Comment:
actually, one more thing:
I'd recommend to use the initializations recommended by jonas & pierre (instead of initializations in the constructor):
```int SimplexType{0}:```

Reply:
Done.